.travis.yml

```diff
@@ -6,6 +6,7 @@ php:
   - 5.4
   - 5.5
   - 5.6
+  - 7.0
```

Comment:
This does not seem to be related to the Symfony 3.0 support. I propose to move it to a separate PR so we have "one PR per issue".